cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -112,6 +124,11 @@ purge() {
 
     echo "Removing snapd catalog cache"
     rm -f /var/cache/snapd/*
+
+    if test -d /etc/apparmor.d; then
+        echo "Removing extra snap-confine apparmor rules"
+        rm -f /etc/apparmor.d/snap.core.*.usr.lib.snapd.snap-confine
```

Comment:
This location depends on `$SNAP_MOUNT_DIR` (i.e. `snap` is `$(systemd-escape $SNAP_MOUNT_DIR)`

Reply:
The correct name is  `$(systemd-escape --path $SNAP_MOUNT_DIR).core.*.usr.lib.snapd.snap-confine`?

Reply:
On second thought, it's like: `$(systemd-escape --path $SNAP_MOUNT_DIR | tr '-' '.').core.*.usr.lib.snapd.snap-confine`